Document tests/pos/patmatch-in-catch.scala
(2:32..2:38) _empty_/Action# -> 1:23
(3:41..3:47) _empty_/Action# -> 1:23
(4:36..4:42) _empty_/Action# -> 1:23
(4:52..4:68) scala/package.RuntimeException#
(6:20..6:26) _empty_/Action# -> 1:23
(6:29..6:35) _empty_/Action# -> 1:23
(8:11..8:17) _empty_/`patmatch-in-catch$package`.driver().(action) -> 6:12
(9:18..9:33) _empty_/CaughtException.
(10:9..10:15) local0 -> 9:34
(11:16..11:35) _empty_/ExceptionNormalExit. -> 3:13
(12:13..12:19) local0 -> 9:34
(14:13..14:16) scala/Predef.`???`().
(15:5..15:11) local1 -> 7:9

